roles/mig_controller_prereqs/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
 mig_controller_location: "{{ workspace }}/mig-controller"
 mig_controller_namespace: "mig"
 mig_controller_sa_name: "mig"
+mig_controller_host_cluster: true
+mig_controller_remote_cluster_online: true
```

Comment:
Thank you for this change, it really allows to test mig-controller deployment, isolated on a single cluster.